docs/source/llms/deployments/index.rst

```diff
@@ -229,8 +229,8 @@ below can be used as a helpful guide when configuring a given endpoint for any n
 +--------------------------+--------------------------+--------------------------+--------------------------+
 |                          | llm/v1/completions       | llm/v1/chat              | llm/v1/embeddings        |
 +==========================+==========================+==========================+==========================+
-| OpenAI                   | - gpt-3.5-turbo          | - gpt-3.5-turbo          | - text-embedding-ada-002 |
-|                          | - gpt-4                  | - gpt-4                  |                          |
+| OpenAI §                 | - gpt-3.5-turbo-instruct | - gpt-3.5-turbo          | - text-embedding-ada-002 |
```

Comment:
What's `§` behind "OpenAI" ?

Reply:
Just a different way of providing a unique footnote :)